### ISSUE TITLE
stuck at page load

### ISSUE THREAD

**Issue by zeitgeistf** [phlntn/emojibuilder#4]
Hi, 

I am a big fan of your tool, it's unfortunate that the site doesn't work anymore. I tried to clone the repo and start the local server myself, but the website stuck at page loading, and I am out of ideas why it's not working, do you mind to take a look at the problem?

what I tried:
1. `git clone https://github.com/phlntn/emojibuilder.git`
2. `cd emojibuilder`
3. `npm install` (received warning saying some of the dependencies have critical risk)
4. `npm start` (server started, and `loading` icon show up on the screen at localhost port 3000), but it stucks there

Thanks,
Shawn

<img width="1376" alt="1" src="https://user-images.githubusercontent.com/11848697/65728303-54310080-e06f-11e9-91c5-eed30b9471b0.png">

**Comment by Revod** [comment]
I got the same problem

**Comment by kud** [comment]
Same here.

**Comment by dfens** [comment]
Same here, this is crucial application for me hlp plz

**Comment by ascheschleier** [comment]
yeah, because there is nothing to load, since there are no emojis included.

**Comment by willow385** [comment]
Same problem

**Comment by rszeberin** [comment]
same for me, does anyone found out the reason for this?

**Comment by beniter** [comment]
@zeitgeistf @rszeberin @kud @willow385 @dfens @Revod 

Hi there! Spent the last month-ish trying to figure out the issue and got some help to figure it out! After creating the assets as outlined in Issue #8 , you need to edit the path created in componentDidMount() in /src/App.js. You need to add "emojibuilder/" without the quotation marks at the front of the definition of the url contained within the promise (given that that's what your master directory is called). That looks like this for me: 

    return Promise.all(assets.map(asset => {
      return new Promise((resolve, reject) => {
        const url = `emojibuilder/${config.assetBasePath}/${asset.name}.png?v4`;
